.eslintrc.json

```diff
@@ -13,17 +13,15 @@
   "overrides": [
     {
       "files": ["*.ts", "*.tsx"],
-      "extends": [
-        "@zendeskgarden/eslint-config/plugins/typescript",
-        "prettier"
-      ],
+      "extends": ["@zendeskgarden/eslint-config/plugins/typescript", "prettier"],
       "rules": {
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/explicit-module-boundary-types": "off",
         "@typescript-eslint/naming-convention": "off",
         "@typescript-eslint/no-explicit-any": "off",
         "logical-assignment-operators": "off",
         "prefer-object-has-own": "off",
+        "react/destructuring-assignment": "off",
```

Comment:
Was this disabled due to the volume of violations? Ideally, this would be a rule that Garden doesn't ignore.

Reply:
Yes that was the reason (to preserve the source mainly) - let me go back and make the changes in the code to remove this.